_episodes/03-computing-pi.md

```diff
@@ -5,21 +5,20 @@ exercises: 30
 questions:
 - "What is the GIL?"
 - "How do I parallelize an elementary program?"
-- "What is vector-based parallelism?"
-- "What is task-based parallelism?"
+- "What is data parallelism?"
```

Comment:
I can find more references to data-parallelism than vector-parallelism, so I changed it. But would be nice to get an 2nd opinion on this.